CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+
+6.0.0-beta.5 / YYYY-MM-DD
+==================
+
+## New features
```

Comment:
If we’re going to have these categories/subtitles, we also need to differentiate bugfixes from improvements. I also _personally_ don’t like the "Other, safer changes" since it doesn’t say what this is—is it a removal of something, a new feature, an improvement?